capsul/pipeline/python_export.py

```diff
@@ -66,21 +66,21 @@ def _write_process(process, pyf, name, enabled, skip_invalid):
                 make_opt.append(fname)
         node_options = ""
         if len(make_opt) != 0:
-            node_options += ", make_optional=%s" % repr(make_opt)
+            node_options += f", make_optional={make_opt!r}"
         if skip_invalid:
             node_options += ", skip_invalid=True"
         print(
-            '        self.add_process("%s", "%s"%s)' % (name, procname, node_options),
+            f'        self.add_process("{name}", "{procname}"{node_options})',
             file=pyf,
         )
 
         # check that sub-nodes enable and plugs optional states are the
         # expected ones
-        todo = [('self.nodes["%s"]' % name, process, proc_copy)]
+        todo = [(f'self.nodes[{name}"]', process, proc_copy)]
```

Comment:
There is a quote missing here:
`f'self.nodes["{name}"]'`
As a matter of fact, tests fail.

Reply:
That's reassuring :+1:

Reply:
Fixed.